registry/registry_test.go

```diff
@@ -148,6 +148,14 @@ func TestGetCipherSuite(t *testing.T) {
 	if err == nil {
 		t.Error("did not return expected error about unknown cipher suite")
 	}
+
+	var insecureCipherSuites = []string{"TLS_RSA_WITH_RC4_128_SHA", "TLS_RSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_ECDSA_WITH_RC4_128_SHA", "TLS_ECDHE_RSA_WITH_RC4_128_SHA", "TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256"}
+	for _, suite := range insecureCipherSuites {
+		_, err = getCipherSuites([]string{suite})
+		if err == nil {
+			t.Errorf("cipher suite %s is not the expected cipher suite", suite)
```

Comment:
that error message is confusing - it should be blocked but isnt should be the message.

Reply:
This is a test, so it's not that important so to speak, but making it clearer would be nice for the maintainers, indeed.

Something like
```
"Unexpected insecure cipher suite: %s"
```

Reply:
Wondering; if TLS 1.3 is used, I think (if I understand correctly) the `CipherSuites` option no longer has an effect  (see https://go.dev/blog/tls-cipher-suites).

Should we not set it if TLS 1.3 is used? (Or produce an error?)

Reply:
Throwing an error to warn the user of the invalid CipherSuites configuration? How about just give a warn log here, and not set it(or just set it - no effect). And keep the registry running.

Reply:
I think warning is the better course of action 🤔